build.gradle

```diff
@@ -1,47 +1,28 @@
 buildscript {
     repositories {
-        mavenLocal()
         mavenCentral()
-        jcenter()
     }
 
     dependencies {
         classpath "org.elasticsearch.gradle:build-tools:${es_version}"
     }
 }
 
+repositories {
+    mavenCentral()
+    mavenLocal()
+    maven {
+        url "https://repo1.maven.org/maven2"
+    }
+}
+
 group = 'org.elasticsearch.plugin'
-version = "${plugin_version}"
```

Comment:
this should not be removed because it creates non-versioned distribution `geopoint-clustering-aggregation.zip` instead of `geopoint-clustering-aggregation-7.17.0.0.zip`